site.deploy.yml

```diff
@@ -65,24 +86,30 @@
         - lamp_installed.changed
         - ansible_deployed.changed
 
+# If we haven't failed by this point, we're ready to prep for
+# service and WordPress deployment.
 - name: Deploy WordPress to defined hosts in the 'wordpress' group
   any_errors_fatal: true
   gather_facts: false
   hosts: wordpress
 
+  # Create '/etc/ansible'.
   pre_tasks:
     - name: Create /etc/ansible
       file:
         path: /etc/ansible
         state: directory
 
+    # Touch '/etc/ansible/wordpress-ultrastack' to mark this host as
+    # being managed by Ansible.
```

Comment:
Not sure this comment adds additional clarification compared to the task `name:`